### ISSUE TITLE
Clicking multiple mouse buttons simultaneously doesn't work in Chrome

### ISSUE THREAD

**Issue by samhed** [novnc/noVNC#1306]
**Describe the bug**
While holding a click with one mouse button and simultaneously clicking with another mouse button will result in that the second mouse click never gets through.

**To Reproduce**
1. Connect using noVNC
2. Monitor mouse events using 'xev' in a terminal on the remote
3. Press the left mouse button but don't release it
4. Click with the right mouse button
5. Release the left mouse button
6. Observe that the right click never got sent to the remote

**Expected behavior**
I expect all mouse clicks to get sent

**Client (please complete the following information):**
 - OS: Fedora 30
 - Browser: Chrome
 - Browser version: 77

**Server (please complete the following information):**
 - noVNC version: master (412d930)
 - VNC server: TigerVNC
 - WebSocket proxy: websockify

**Additional context**
It works fine on Firefox

**Comment by mmiscool** [comment]
Is there any word on this one possibly being fixed in the live git but not the release yet?

**Comment by samhed** [comment]
This isn't something anyone has had the time to look at yet.

**Comment by mmiscool** [comment]
If you might point me source file that handles this sort of thing I could try and work it out.

**Comment by CendioOssman** [comment]
The relevant file is `core/input/mouse.js`. Any debugging you can do would be helpful.

**Comment by CendioOssman** [comment]
I've debugged this, and it is unfortunately some bug in Chrome. I've reported it to them here:

https://bugs.chromium.org/p/chromium/issues/detail?id=1098276

For now we'll wait and see how they respond.

**Comment by CendioHalim** [comment]
This should now be fixed as of #1921?

**Comment by CendioOssman** [comment]
Still broken for me. Does it work for you?

**Comment by CendioHalim** [comment]
Ah, looks like the first mouse press/release after an initial press don't produce any events, but subsequent button presses do.

**Comment by CendioOssman** [comment]
Indeed. Which is a bug, I'd say, because the capture is dropped at that point. You can see that we no longer get any move events after leaving the viewport. Which we should.
